rts/motoko-rts/src/gc/generational/remembered_set.rs

```diff
@@ -82,6 +82,8 @@ impl RememberedSet {
         if is_null_ptr_value(value) || value.get_raw() == self.fast_cache.get_raw() {
             return;
         }
+        self.fast_cache = Value::from_raw(value.get_raw());
```

Comment:
```suggestion
        self.fast_cache = value;
```
Or are the conversions doing something?

Reply:
I removed the cache as there is no relevant performance gain. It was useful for the remembered log (to eliminate a lot of duplicates)